tests/general/KeywordBearTest.py

```diff
@@ -14,3 +22,99 @@
     invalid_files=("test line todo",
                    "test line warNING"),
     settings={"keywords": "todo, warning"})
+
+
+class KeywordBearDiffTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section("")
+        self.section.append(Setting('language', 'python3'))
+        self.section.append(Setting('keywords', '\# TODO, \#todo'))
+        self.python_uut = KeywordBear(self.section, Queue())
+
+        self.annotation_bear_result_type = namedtuple('result', ['contents'])
+        self.text = ['# todo 123']
```

Comment:
as you reset the test-`text` everwhere except in two cases, I would move this out of `setUp()` and just do below
```python
text = ['# todo 123']
# tests...
```